source/Shuffle.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2017 Akshay Agarwal <agarwal.akshay.akshay8@gmail.com>
+ * All rights reserved. Distributed under the terms of the MIT license.
+ */
+#ifndef SHUFFLE_H
+#define SHUFFLE_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+
+void Swap(int* a, int* b);
```

Comment:
Any reason for doing this with your own code rather than using http://en.cppreference.com/w/cpp/algorithm/random_shuffle ?